### PR TITLE
Initial commit of cron,checking script

### PR DESCRIPTION
**Pull request by robotmaxtron**

### CODE REVIEW

File: scripts/monitoring/cron-send-user-count.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+COUNT=$(oca get --no-headers users | wc -l)
+echo
+echo "Number of users : $COUNT"
```
Comment (mmahut) [inline scripts/monitoring/cron-send-user-count.sh]: Can we get output from this script only if --verbose/-v is passed? That way we do not get spammed from cron's output.

File: scripts/openshift-tools-scripts.spec [hunk 2]
```diff
@@ -23,6 +23,7 @@ cp -p monitoring/ops-zagg-client.py %{buildroot}/usr/bin/ops-zagg-client
 cp -p monitoring/ops-zagg-processor.py %{buildroot}/usr/bin/ops-zagg-processor
 cp -p monitoring/ops-zagg-heartbeater.py %{buildroot}/usr/bin/ops-zagg-heartbeater
 cp -p monitoring/cron-send-process-count.sh %{buildroot}/usr/bin/cron-send-process-count
+cp -p monitoring/cron-send-user-count.sh %{buildroot}/usr/bin/cron-send-user-count.sh
```
Comment (kwoodson) [inline scripts/openshift-tools-scripts.spec]: Remove .sh on the '%{buildroot}/usr/bin/cron-send-user-count.sh'

File: docker/oso-rhel7-zagg-client/root/config.yml [hunk 3]
```diff
@@ -45,6 +45,9 @@
     - name: send openshift-master process count
       minute: "*/2"
       job: /usr/bin/cron-send-process-count '^/usr/bin/openshift start master' openshift.master.process.count
+    - name: send openshift-master user count every 3 minutes
+      minute: "*/3"
+      job: /usr/local/bin/cron-send-user-count.sh
```
Comment (kwoodson) [inline docker/oso-rhel7-zagg-client/root/config.yml]: Remove .sh on this script name.

File: docker/oso-rhel7-zagg-client/root/config.yml [hunk 4]
```diff
@@ -45,6 +45,9 @@
     - name: send openshift-master process count
       minute: "*/2"
       job: /usr/bin/cron-send-process-count '^/usr/bin/openshift start master' openshift.master.process.count
+    - name: send openshift-master user count every 3 minutes
+      minute: "*/3"
```
Comment (kwoodson) [inline docker/oso-rhel7-zagg-client/root/config.yml]: Suggestion:  Update this check to 4-6 hours.

File: docker/oso-rhel7-zagg-client/root/config.yml [hunk 5]
```diff
@@ -45,6 +45,9 @@
     - name: send openshift-master process count
       minute: "*/2"
       job: /usr/bin/cron-send-process-count '^/usr/bin/openshift start master' openshift.master.process.count
+    - name: send openshift-master user count every 6 hours
+      minute: "0 */6"
```
Comment (kwoodson) [inline docker/oso-rhel7-zagg-client/root/config.yml]: minute: 0
hour: */6

**Comment by robotmaxtron** [comment]
okay to test

**Pull request closed without merging**